vnext/Shared/Modules/FileReaderModule.h

```diff
@@ -48,7 +50,27 @@ class FileReaderModule : public facebook::xplat::module::CxxModule {
 #pragma endregion CxxModule
 
  private:
-  std::weak_ptr<IBlobPersistor> m_weakBlobPersistor;
+  std::shared_ptr<IFileReaderResource> m_resource;
+};
+
+REACT_MODULE(FileReaderTurboModule, L"FileReaderModule")
+struct FileReaderTurboModule {
+  REACT_INIT(Initialize)
```

Comment:
#include NativeFileReaderModuleSpec.g.h

Then add 
using ModuleSpec = ReactNativeSpecs::FileReaderModuleSpec;

That will verify that you are correctly implementing the interface and break if the interface changes.